utils/build-presets.ini

```diff
@@ -859,7 +859,6 @@ install-lldb
 install-llbuild
 install-swiftpm
 install-swift-driver
-install-swiftsyntax
```

Comment:
Could just leave this in since the produce is changed. There's a lot of these that would need to be removed anyway.